blackbelt/tasks.py

```diff
@@ -15,12 +15,15 @@ def list_commands(self, ctx):
         return rv
 
     def get_command(self, ctx, name):
-        ns = {}
-        fn = os.path.join(plugin_folder, name + '.py')
-        with open(fn) as f:
-            code = compile(f.read(), fn, 'exec')
-            eval(code, ns, ns)
-        return ns['cli']
+        try:
+            ns = {}
+            fn = os.path.join(plugin_folder, name + '.py')
+            with open(fn) as f:
+                code = compile(f.read(), fn, 'exec')
+                eval(code, ns, ns)
+            return ns['cli']
+        except:
```

Comment:
@abtris that is because I spend too much time in javascript; `Exception` that were there before edit is correct:

`except Exception:`